courses/management/commands/update_courses.py

```diff
@@ -12,7 +12,7 @@
 from easy_thumbnails.exceptions import InvalidImageFormatError
 from easy_thumbnails.files import get_thumbnailer
 
-from courseware.courses import get_permission_for_course_about, get_course_with_access
+from courseware.courses import get_permission_for_course_about, get_course
```

Comment:
It seems to me we could use the `get_course` method from the modulestore, which would allow us to not make any import from the courseware, which is an LMS-only application.

Just write:

    course = modulestore().get_course(self.course_descriptor.id)